logexporter/cmd/main.go

```diff
@@ -77,8 +78,14 @@ func checkConfigValidity() error {
 	if _, err := os.Stat(*gcloudAuthFilePath); err != nil {
 		return fmt.Errorf("Could not find the gcloud service account file: %v", err)
 	} else {
+		glog.Infof("Running gcloud auth activate-service-account --key-file=%s\n", *gcloudAuthFilePath)
 		cmd := exec.Command("gcloud", "auth", "activate-service-account", "--key-file="+*gcloudAuthFilePath)
-		if err := cmd.Run(); err != nil {
+		var stderr, stdout bytes.Buffer
+		cmd.Stderr, cmd.Stdout = &stderr, &stdout
+		err = cmd.Run()
+		glog.Infof("Stdout:\n%s\n", stdout.String())
+		glog.Infof("Stderr:\n%s\n", stderr.String())
```

Comment:
To avoid polluting all logs, maybe we should log stdout/stderr only if err != nil ?

Reply:
In case of successful run, stderr is empty and stdout contains only one line: `Activated service account credentials for: [pr-kubekins@kubernetes-jenkins-pull.iam.gserviceaccount.com]`
Given how the logexporter log looks currently (e.g. https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-scalability/23037/artifacts/e2e-big-minion-group-16fq/logexporter-f8wm9.log) I don't think this change will add much pollution. 

What's more, I think it might be useful to have these logs always, e.g. if `gcloud auth` starts outputting some warning, these logs may help us to debug or catch an issue before it starts failing.

Let me know what you think.

Reply:
fair enough